kraken-app/kraken-app-portal/src/components/ApiMapping/ApiCard.tsx

```diff
@@ -25,14 +27,17 @@ export function ApiCard({
       data-testid="useCase"
       align="center"
       gap={10}
+      style={style}
       className={classNames(className, styles.rowBorder)}
       onClick={onClick}
     >
       {prefix}
       <RequestMethod method={apiInstance?.method} />
-      <span data-testid="apiPath">
-        {"/" + apiInstance.path.split("/").slice(-2).join("/")}
-      </span>
+      <Tooltip title={apiInstance.path}>
+        <span data-testid="apiPath">
+          {".../" + apiInstance.path.split("/").slice(-2).join("/")}
+        </span>
```

Comment:
I'm aware of that component, but I managed not to use it because I will have to restyle and it make the code untestable. Perhaps I will write a utility function the trim the path and use it in both TrimmedPath component as well as other places.